airflow/www_rbac/views.py

```diff
@@ -207,6 +206,20 @@ def get_int_arg(value, default=0):
             unfiltered_webserver_dags = [dag for dag in dagbag.dags.values() if
                                          not dag.parent_dag]
 
+        if not filter_dag_ids:
+            orm_dags = {}
+            unfiltered_webserver_dags = []
```

Comment:
I think the `else:` would effectively take care of that as an effective: `DM.dag_id.in_([])`

Reply:
updated